src/store/cart-store.js

```diff
@@ -1,26 +1,54 @@
 import { Store } from 'consus-core/flux';
 import StudentStore from './student-store';
+import { checkOutItems } from '../lib/api-client';
 
 let items = [];
 
+let timer = null;
+let isOnTimer = false;
```

Comment:
`isOnTimer === (timer !== null)` should always be `true`, right? Seems redundant.